pkgs/applications/networking/mailreaders/hasmail/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, pkgconfig
+, gobject-introspection
+, pango
+, cairo
+, gtk2
```

Comment:
The program doesn't support `gtk3`?

Reply:
I get the error:

```
# pkg-config --cflags  -- gdk-2.0 gthread-2.0
Package gdk-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gdk-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gdk-2.0' found
pkg-config: exit status 1
```

And according to `nix-locate gdk-2.0.pc`, only GTK 2 provides this file.